bindings/go/scip/symbol_formatter.go

```diff
@@ -8,42 +8,71 @@ import (
 // Excluding parts of the symbol can be helpful for testing purposes. For example, snapshot tests may hardcode
 // the package version number so it's easier to read the snapshot tests if the version is excluded.
 type SymbolFormatter struct {
-	OnError               func(err error) error
-	IncludeScheme         func(scheme string) bool
-	IncludePackageManager func(manager string) bool
-	IncludePackageName    func(name string) bool
-	IncludePackageVersion func(version string) bool
-	IncludeDescriptor     func(descriptor string) bool
+	OnError                 func(err error) error
+	IncludeScheme           func(scheme string) bool
+	IncludePackageManager   func(manager string) bool
+	IncludePackageName      func(name string) bool
+	IncludePackageVersion   func(version string) bool
+	IncludeDescriptorString func(descriptor string) bool
+	IncludeDescriptor       func(descriptor *Descriptor) bool
```

Comment:
This seems to be a breaking change. In general, I think it's desirable to avoid breaking changes since it's tiresome to consume bindings that regularly change their API. In this case, we could keep the old filter and add a new `IncludeParsedDescriptor(...)` filter

Reply:
Not a blocking comment, just an observation. I don't think these bindings have high usage outside of Sourcegraph. Still, my general rule of thumb for public APIs is to never break them unless you absolutely have to.

Reply:
`IncludeDescriptorString` is the old `IncludeDescriptor`. Renamed it, see new diff.